runtime/config.go

```diff
@@ -0,0 +1,102 @@
+package runtime
+
+import (
+	"encoding/json"
+	"os"
+
+	typesCons "github.com/pokt-network/pocket/consensus/types"
+	typesP2P "github.com/pokt-network/pocket/p2p/types"
+	typesPers "github.com/pokt-network/pocket/persistence/types"
+	"github.com/pokt-network/pocket/shared/modules"
+	typesTelemetry "github.com/pokt-network/pocket/telemetry"
+	typesUtil "github.com/pokt-network/pocket/utility/types"
+)
+
+var _ modules.ConsensusConfig = &Config{}
+var _ modules.P2PConfig = &Config{}
+var _ modules.PersistenceConfig = &Config{}
+var _ modules.TelemetryConfig = &Config{}
+var _ modules.UtilityConfig = &Config{}
+
+type Config struct {
```

Comment:
I really prefer if this was an exported interface and not an exported structure

Reply:
the renamed `runtimeConfig` now implements `modules.Config`.

runtime package exports the constructor and the `BaseConfig`

https://github.com/pokt-network/pocket/pull/235/commits/1924e0e7e3c6988d13cb0b4a2a4edc5c07c64f2b#diff-22e2eb4a42003f34d95bacbe4bf7dfa8d3ddd6ec3eb66c5ed611b5b6b72de26bR14-R23